RoboFile.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use DigipolisGent\Robo\Symfony\RoboFileBase;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Auth\AbstractAuth;
+
+class RoboFile extends RoboFileBase
+{
+    protected function postSymlinkTask($worker, AbstractAuth $auth, $remote)
+    {
+        $currentProjectRoot = $remote['currentdir'] . '/..';
+        $collection = $this->collectionBuilder();
+        $parent = parent::postSymlinkTask($worker, $auth, $remote);
+        if ($parent) {
+            $collection->addTask($parent);
+        }
+        $collection->taskSsh($worker, $auth)
+            ->remoteDirectory($currentProjectRoot, true)
+            ->exec('composer symfony-scripts');
```

Comment:
Composer install is executed on the build server, the symlinks on the test/qa/prod servers. problem is dat the symfony scripts load config, which loads parameters.yml, which isn't available until after the symlinks.